qcodes/tests/dataset/test_datasaver.py

```diff
@@ -4,26 +4,21 @@
 import qcodes as qc
 from qcodes.dataset.measurements import DataSaver
 from qcodes.dataset.sqlite_base import connect, init_db
+from qcodes.dataset.database import initialise_database
 
 CALLBACK_COUNT = 0
 CALLBACK_RUN_ID = None
 CALLBACK_SNAPSHOT = None
 
-# These fixture can't be imported from test_dataset_basic because Codacy/PR Quality Review will think they are unused.
+# These fixture can't be imported from test_dataset_basic because
```

Comment:
This is a very bad argument. We should never write uglier code to make a linter happy

Reply:
But I know its not related to this pr